Platform/OpenCanopy/Views/BootPicker.c

```diff
@@ -1176,15 +1177,19 @@ BootPickerViewInitialize (
     InternalBootPickerExitLoop,
     GuiContext
     );
+  DrawContext->Scale = GuiContext->Scale;
```

Comment:
Was it not set before?

Reply:
There are no other assignments to this variable

Reply:
Yes, there are no other assignments.